CHANGELOG.md

```diff
@@ -12,10 +13,6 @@
 # v46.1.0 (27/01/2021)
 
 - **[NEW]** Add `PaypalIcon`
-  <<<<<<< HEAD
```

Comment:
nit: rebase issue

Reply:
yep, rebased already

Reply:
It's my fault, It will be fix on this https://github.com/blablacar/ui-library/pull/1070